### DISCUSSION TITLE
Terraform provider

### DISCUSSION THREAD

**Discussion by tomwphillips** [openfga/community#328] [Ideas]
We'd like to be able to create stores and authorization models via Terraform. Are there plans for a Terraform provider? If not, I might be able to create one.

**Comment by aaguiarz** [comment]
Hi! We discussed it, but it's not planned yet.

I've seen this one https://github.com/cysp/terraform-provider-openfga, not sure if it's similar to what you'd need.

**Reply by carlreid** [reply]
Would `cysp/terraform-provider-openfga` be adopted by OpenFGA if it came to supporting Terraform? Would like to avoid needing to migrate to a more official approach in the future if possible.

**Reply by aaguiarz** [reply]
Hi @carlreid 

We are planning to use this one as an "official" provider https://github.com/mauriceackel/terraform-provider-openfga

cc @mauriceackel

**Reply by carlreid** [reply]
Ah great, thanks for the speedy response and heads up 😊 Will look into that provider, thanks Maurice!

**Reply by dyeam0** [reply]
Hi @carlreid and @tomwphillips. Maurice's Terraform provider has been moved into OpenFGA: https://github.com/openfga/terraform-provider-openfga!

**Reply by carlreid** [reply]
Thanks for the ping @dyeam0, that's really great to hear 👍
